tests/test_qc_metrics.py

```diff
@@ -203,7 +203,7 @@ def adata_mito():
 
 skip_if_adata_0_11_4 = pytest.mark.skipif(
     Version(version("anndata")) >= Version("0.11.4.dev2"),
```

Comment:
Maybe I'm unfamiliar but it seems like 0.11.5>0.11.4 but that this test should not be skipped in that case?

Reply:
good point! I fixed versions in https://github.com/scverse/anndata/pull/1831

```suggestion
skip_if_adata_0_12 = pytest.mark.skipif(
    Version(version("anndata")) >= Version("0.12.0.dev0"),
```